src/data/actions/assets.js

```diff
@@ -188,11 +184,6 @@ export const pageUpdate = (page, courseDetails) =>
 
     return dispatch(getAssets(parameters, courseDetails)).then((responseAction) => {
       if (requestFailed(responseAction)) {
-        // react-paginate does not force the page state via the forcePage prop when
-        // the previous and next props for forcePage are the same; therefore, we
-        // have to reset the page state after a failed click in order to be able
-        // to reset the page
-        dispatch(resetPageState());
```

Comment:
Does the Paragon component handle this failure more gracefully?

Reply:
Correct! I tested whether this use-case were necessary using network throttling features: I throttled to slow to make the request slow enough to catch, and then turned off the network while it was in-flight to trigger this code path. After deletion, there was no difference in behavior without it.